lib/chef/resource/file.rb

```diff
@@ -115,6 +117,18 @@ def manage_symlink_source(arg=nil)
           :kind_of => [ TrueClass, FalseClass ]
         )
       end
+
+      def verify(command=nil, opts={}, &block)
+        if ! (command.nil? || [String, Symbol].include?(command.class))
+          raise ArgumentError, "verify requires either a string, symbol, or a block"
```

Comment:
I'm a bit confused but I might be misunderstanding.  Without arguments or a block, what is there to validate?  In that case, this method returns the current set of validations, much like the other set_or_return style methods in this class.